src/hyperion/experiment_plans/robot_load_then_centre_plan.py

```diff
@@ -130,7 +130,7 @@ def take_robot_snapshots(oav: OAV, webcam: Webcam, directory: Path):
 def prepare_for_robot_load(composite: RobotLoadThenCentreComposite):
     yield from bps.abs_set(
         composite.aperture_scatterguard,
-        AperturePosition.ROBOT_LOAD,
+        (ApertureInOut.OUT, None),
         group="prepare_robot_load",
     )
```

Comment:
We should do a set to `(OUT, SMALL)` small after this but will need to think about when to wait for it